addOns/pscanrulesBeta/src/test/java/org/zaproxy/zap/extension/pscanrulesBeta/SiteIsolationScanRuleTest.java

```diff
@@ -95,6 +114,22 @@ void shouldNotRaiseAlertGivenSiteIsIsolatedWhenSuccessAndIdentifiedByCustomPage(
         assertThat(alertsRaised, hasSize(0));
     }
 
+    @Test
+    void shouldNotRaiseAlertGivenSiteIsNotIsolatedWhenFailureIdentifiedByCustomPage()
+            throws Exception {
+        // Given
+        HttpMessage msg = new HttpMessage();
+        msg.setRequestHeader("GET / HTTP/1.1");
+        msg.setResponseHeader("HTTP/1.1 200 OK\r\n" + "Content-Type: text/html");
+        given(passiveScanData.isPage200(any())).willReturn(false);
```

Comment:
Same

Reply:
Removed this whole test, as this was meant to trigger a path that never existed (see earlier comment)

Reply:
Then set 404 instead of 200 🙂

Reply:
Seems there's a rendering issue with the linked page (on safari anyway). I'll dig into that... accessed via shows a chain link icon but no text.